create_hapi_k8s.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
```

Comment:
Thoughts on adding the following to exit if any command fails?

```sh
set -euo pipefail
```

> https://bertvv.github.io/cheat-sheets/Bash.html#writing-robust-scripts-and-debugging

Reply:
will do.